frontends/torch-frontend/third_party/patches/communication_op.patch

```diff
@@ -70,15 +108,59 @@ index 00000000..d8a58997
 +    }
 +  }];
 +}
-diff --git a/include/torch-mlir/Dialect/Torch/IR/TorchOps.td b/include/torch-mlir/Dialect/Torch/IR/TorchOps.td
-index c86244f5..196e3a91 100644
---- a/include/torch-mlir/Dialect/Torch/IR/TorchOps.td
-+++ b/include/torch-mlir/Dialect/Torch/IR/TorchOps.td
-@@ -23,6 +23,7 @@ class Torch_Op<string mnemonic, list<Trait> traits = []>
- }
- 
- include "torch-mlir/Dialect/Torch/IR/GeneratedTorchOps.td"
-+include "torch-mlir/Dialect/Torch/IR/TorchFunctionalCommunicationOps.td"
- 
- //===----------------------------------------------------------------------===//
- // TorchScript `torch.nn.Module` object instantiation ops.
++
++def Torch_C10dFunctionalIsendOp : Torch_Op<"c10d_functional.isend", [
```

Comment:
Maybe add send/recv (synchronous version of isend/irecv) collective operators here also?